apps/console/src/extensions/components/users/pages/users.tsx

```diff
@@ -445,27 +448,117 @@ const UsersPage: FunctionComponent<UsersPageInterface> = (
         />
     );
 
+    const addUserDropdownTrigger: ReactElement = (
+        <PrimaryButton
+            data-componentid={ `${ componentId }-add-user-button` }
+            data-testid={ `${ testId }-add-user-button` }
+        >
+            <Icon name="add"/>
+            { t("extensions:manage.users.buttons.addUserBtn") }
+            <Icon name="dropdown" className="ml-3 mr-0"/>
+        </PrimaryButton>
+    );
+
+    const getAddUserOptions = () => {
+        const options: DropdownItemProps = [
+            {
+                "data-componentid": `${ componentId }-admin-add-user-dropdown-item`,
+                "data-testid": `${ testId }-admin-add-user-dropdown-item`,
```

Comment:
Check the other places as well.
```suggestion
                "data-testid": `${ testId }-add-user-dropdown-item`,
```